tools/jslint/jslint.js

```diff
@@ -607,7 +607,10 @@ var JSLINT = (function () {
             use_object: "Use the object literal notation {}.",
             use_or: "Use the || operator.",
             use_param: "Use a named parameter.",
-            used_before_a: "'{a}' was used before it was defined.",
+            used_before_a: "'{a}' was used before it was defined. (If you " +
+                " have added a new module, you need to add it to the whitelist " +
```

Comment:
Looks like this will result in a redundant space; but otherwise lgtm.  I can just fix that and merge when I get a chance.  Thanks @karamcnair !